constraint/Makefile

```diff
@@ -33,8 +40,18 @@ vet:
 	go vet ./pkg/...
 
 # Generate code
-generate:
+generate: .output/buildutil
 ifndef GOPATH
 	$(error GOPATH not defined, please define GOPATH. Run "go help gopath" to learn more about GOPATH)
 endif
-	go generate ./pkg/...
+	export GOROOT=$(go env root)
+	GO111MODULE=off go generate ./pkg/...
```

Comment:
Getting the following on my mac:

```
export GOROOT=
GO111MODULE=off go generate ./pkg/...
pkg/apis/apis.go:17: running "../../.output/buildutil/deepcopy-gen": fork/exec ../../.output/buildutil/deepcopy-gen: exec format error
pkg/core/templates/constrainttemplate_types.go:17: running "../../../.output/buildutil/deepcopy-gen": fork/exec ../../../.output/buildutil/deepcopy-gen: exec format error
make[1]: *** [generate] Error 1
make: *** [build-constraint] Error 2
```

We can either make sure everything is built/generated in a container or make sure things run natively on mac. WDYT?

Reply:
Fixed to run natively